bpf/process/types/bpfmap.h

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: GPL-2.0
+/* Copyright Authors of Cilium */
+
+#ifndef __BPFMAP_H__
+#define __BPFMAP_H__
+
+struct bpf_map_info_type {
+	__u32 map_type;
+	__u32 key_size;
+	__u32 value_size;
+	__u32 max_entries;
+	char map_name[16U];
```

Comment:
Could you please #define the 16U to something like BPF_MAP_NAME_LEN? and use it everywhere, same goes for go user space part?